markdown/manual.md

```diff
@@ -18,25 +18,24 @@ In our work, we will strive to uphold these values:
 
-We are committed to providing equal employment opportunities to qualified individuals with disabilities.  This may include providing reasonable accommodation where appropriate in order for an otherwise qualified individual to perform the essential functions of the job.
+We are committed to providing equal employment opportunities to qualified individuals with disabilities. This may include providing reasonable accommodation where appropriate in order for an otherwise qualified individual to perform the essential functions of the job.
 
 ### Harassment, Bullying, Violence in the Workplace Policy
 We prohibit harassment of one employee by another employee, supervisor or third party for any reason based on a protected class including but not limited to: veteran status, uniform member status, race, color, religion, sex, national origin, sexual orientation, physical or mental disability genetic information or any other protected class under federal, state or local law, *as well as* gender identity, parental status, marital status, political affiliation, gender expression, mental illness, socioeconomic status or background, neuro(a)typicality, physical appearance, body size, or clothing. Harassment of third parties by our employees is also prohibited.
 
 The following are considered a protected class: race; religious creed; color; national origin; ancestry; physical disability; mental disability; medical condition, including genetic characteristics; marital status; sex; pregnancy, childbirth or related medical conditions; actual or perceived gender; gender identity; sexual orientation; civil air patrol membership; service in the military forces and age (40 or over). Included in the definition of each protected category is the perception of membership in a protected category and an individual's association with an actual or perceived member of a protected category.
 
-The purpose of this policy is to ensure that in the workplace, no employee harasses another for any reason or in any manner. The conduct prohibited by this policy includes conduct in any form including but not limited to e-mail, voice mail, chat rooms, social networking, internet use or history, text messages, pictures, images, writings, words or gestures.
+The purpose of this policy is to ensure that all employees shall be from harassment of any kind.  The conduct prohibited by this policy includes conduct in any form including but not limited to e-mail, voice mail, chat rooms, social networking, internet use or history, text messages, pictures, images, writings, words or gestures.
```

Comment:
"shall be from harassment" should be "shall be *free* from harassment", I assume.